com.unity.netcode.gameobjects/Runtime/Serialization/FastBufferWriter.cs

```diff
@@ -777,6 +777,11 @@ public unsafe void CopyFrom(FastBufferWriter other)
             }
         }
 
+        // These structs enable overloading of WriteValue with different generic constraints.
+        // The compiler's actually able to distinguish between overloads based on generic constraints.
+        // But at the bytecode level, the constraints aren't included in the method signature.
+        // By adding a second parameter with a defaulted value, the signatures of each generic are different,
+        // thus allowing overloads of methods based on the first parameter meeting constraints.
```

Comment:
Didn't intend it to be since it's referring to a whole section and not a specific struct or method.

I wasn't really sure what to do with xmldocs for this section because the methods are one line each and adding a 5 line xmldoc for each one seemed overkill.